nexus/src/sagas.rs

```diff
@@ -1289,6 +1289,78 @@ async fn sdc_regions_ensure(
 
     let block_size = datasets_and_regions[0].1.block_size;
 
+    // If requested, back disk by image
+    let osagactx = sagactx.user_data();
+    let params = sagactx.saga_params();
+    let opctx = OpContext::for_saga_action(&sagactx, &params.serialized_authn);
+    let read_only_parent: Option<
+        Box<sled_agent_client::types::VolumeConstructionRequest>,
+    > = if let Some(image_id) = params.create_params.image_id {
+        let log = osagactx.log();
+        warn!(log, "grabbing image {}", image_id);
+
+        // TODO: support project images too
```

Comment:
Are we planning on distinguishing global/project scope in the create parameters (like, via enum or something)?

Reply:
Addressed by 0477683b